cobbler.spec

```diff
@@ -384,6 +384,40 @@ chgrp %{apache_group} %{_sysconfdir}/cobbler/settings.yaml
 %else
 %post
 %systemd_post cobblerd.service
+
+if [ $1 -gt 1 ]; then
+    if [ -f /etc/susemanager-release ] || [ -f /etc/uyuni-release ]; then
+        if [ -f /etc/cobbler/settings.yaml.rpmnew ] && [ ! -f /etc/cobbler/settings.d/zz-uyuni.settings ]; then
+            # The /etc/cobbler/settings.yaml.backup should be existing unless explicitely removed.
+            if [ ! -f /etc/cobbler/settings.yaml.backup ]; then
```

Comment:
`yaml.safe_load` can still return `None` when the file exists, but is empty.  We check whether file exists, but not whether it's empty.

Would it be worth checking this here? (or later in Python, whichever is easier)